ui/page/wallets/wallet_page.go

```diff
@@ -730,9 +761,16 @@ func (pg *WalletPage) layoutCollapsibleHeader(gtx layout.Context, listItem *wall
 					return pg.Theme.Body1(listItem.wal.Name).Layout(gtx)
 				}),
 				layout.Rigid(func(gtx C) D {
+					var txt decredmaterial.Label
 					if len(listItem.wal.EncryptedSeed) > 0 {
-						txt := pg.Theme.Caption(values.String(values.StrNotBackedUp))
+						txt = pg.Theme.Caption(values.String(values.StrNotBackedUp))
 						txt.Color = pg.Theme.Color.Danger
+					}
+					if listItem.wal.IsAccountMixerActive() {
+						txt = pg.Theme.Caption("Mixing...")
+						txt.Color = pg.Theme.Color.GrayText2
+					}
+					if (txt != decredmaterial.Label{}) {
```

Comment:
Why do we need to replace the backup text with the `Mixing` text? the can both exist side by side.

Reply:
from the mobile UI in the issue here https://github.com/planetdecred/godcr/issues/796, that was implemented as per morphymore's design.

Also there is a giant card stating the same information on the wallet, hence the information isn't lost
![Screenshot from 2022-03-21 15-46-10](https://user-images.githubusercontent.com/25265396/159286290-7ff67361-d891-4483-a8ac-c8e98810643d.png)

it's makes it unnecessary or not mandatory to add that information side by side with the mixing text as the mixing text is temporary

Reply:
alright would implement side by side

Reply:
This isnt resolved yet 
<img width="581" alt="image" src="https://user-images.githubusercontent.com/27733432/159507033-be5458d9-6a29-4c11-a84f-efe37d6fcad3.png">